identify/extensions.py

```diff
@@ -223,21 +225,26 @@
     '.bash_profile': EXTENSIONS['bash'],
     '.bashrc': EXTENSIONS['bash'],
     '.bowerrc': EXTENSIONS['json'] | {'bowerrc'},
+    '.browserslintrc': {'text', 'browserslintrc'},
```

Comment:
there's still a typo here

Reply:
also is this file json?

Reply:
ah no it's not, I read their docs wrong

Reply:
Sorry about that, fixed the `txt` -> `text` one but not the name of the file. Fixed now, thanks.